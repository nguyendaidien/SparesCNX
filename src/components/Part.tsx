import React from 'react'
import { Grid, Card, Typography } from '@material-ui/core'

const Part = ({
  name,
  id,
  status
}: {
  name: string
  id: number
  status: string
}) => (
    <Grid container sm={4} xs={12}>
      <Card
        className={'view-card-header'}
        style={{ marginTop: 20, width: '100%' }}
      >
        <Typography>
          <b>ID</b>: {id}
        </Typography>
        <Typography>
          <b>Name</b>: {name}
        </Typography>
        <Typography>
          <b>Status</b>: {status}
        </Typography>
      </Card>
    </Grid>
  )

export default Part
