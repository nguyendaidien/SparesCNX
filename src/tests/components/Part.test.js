import React from 'react';
import { shallow } from 'enzyme';
import Part from '../../components/Part';

test('should test Part component', () => {
    const wrapper = shallow(<Part />);
    expect(wrapper).toMatchSnapshot();
});