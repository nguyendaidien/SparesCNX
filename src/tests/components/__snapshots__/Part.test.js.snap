// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Part component 1`] = `ShallowWrapper {}`;
